component/pom.xml

```diff
@@ -49,21 +49,24 @@
             <artifactId>commons-math3</artifactId>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>test</scope>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.siddhi</groupId>
+            <artifactId>siddhi-annotations</artifactId>
+            <version>4.0.0-SNAPSHOT</version>
```

Comment:
You should not use versions here! they should always be in the parent poms properties section